app/navigation/FirstPage.scala

```diff
@@ -19,14 +19,6 @@ package navigation
 import models.NormalMode
 import play.api.mvc.Call
 
-sealed trait FirstPage {
-  def call: Call
-}
-
-class V1FirstPage extends FirstPage {
-  override def call: Call = controllers.routes.EoriNumberController.onPageLoad(NormalMode)
-}
-
-class V2FirstPage extends FirstPage {
-  override def call: Call = controllers.routes.GuaranteeReferenceNumberController.onPageLoad(NormalMode)
+class FirstPage {
+  def call: Call = controllers.routes.GuaranteeReferenceNumberController.onPageLoad(NormalMode)
```

Comment:
Seems silly to still have this - can we just add the controller route directly to `StartController`?